plugin/trino-faker/src/test/java/io/trino/plugin/faker/TestFakerQueries.java

```diff
@@ -402,4 +402,12 @@ String columnSchema()
             return "%s %s NOT NULL%s".formatted(name, type, propertiesSchema.isEmpty() ? "" : " WITH (%s)".formatted(propertiesSchema));
         }
     }
+
+    @Test
+    void testRenameTable()
+    {
+        assertUpdate("CREATE TABLE faker.default.original_table (id INTEGER, name VARCHAR)");
+        assertUpdate("ALTER TABLE faker.default.original_table RENAME TO faker.default.renamed_table");
```

Comment:
It's a good starting point, but would be nice to assert:
* The table gets renamed, not copied - the original table no longer exists.
* You cannot rename a table to one that already exists.